src/common/exceptions/custom-bad-request-exception.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export class CustomBadRequestException extends HttpException {
+  constructor(message: string, errorCode: string) {
+    super(
+      {
+        success: false,
+        errorCode,
```

Comment:
```suggestion
        errorCod: ErrorCodes.ExistedUser
```

Should add enum where value is number